src/main/java/seedu/address/model/person/Person.java

```diff
@@ -27,10 +28,11 @@ public class Person {
      * Constructs a {@code Person} class when first initialized with add command.
      * // Todo: Note that this is a temporary design and subject to update in future iteration
      */
-    public Person(Name name, Phone phone, Email email, Address address) {
-        requireAllNonNull(name, phone, email, address);
+    public Person(Name name, Phone phone, NOKPhone nokPhone, Email email, Address address) {
```

Comment:
The current `addCommand` makes use of the previous constructor with four arguments: `name`, `phone`, `email`, and `address`. Adding the new argument `nokPhone` has resulted in the code failing to run.